### PR TITLE
[docs] - RFC: add a Learning More section to the asset tutorial [CON-17]

### PR DESCRIPTION
**Pull request by sryza**
The asset tutorial will likely eventually become Dagster's main tutorial. In our current main tutorial, we have Advanced pages that correspond to the pages we have in the Concepts section. Those pages are poorly maintained. Writing a new set of these pages for the asset tutorial is a daunting task that would be very difficult to complete in the 0.15.0 timeframe.

This proposes moving to a model where we instead link out to the Concept pages and explain how they relate to what was covered in the basic tutorial.

### CODE REVIEW

File: docs/content/guides/dagster/asset-tutorial/learning-more.mdx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: Learning More | Dagster
+description: You might discover that your needs start to outgrow the basics covered in the tutorial. This page offers a survey of Dagster's full capabilities and where to learn about each of them in docs.
+---
+
+# Learning More
```
Comment (erinkcochran87) [inline docs/content/guides/dagster/asset-tutorial/learning-more.mdx:6]: My initial impressions of this were:

1. Wow, this is really thorough!
2. This is a bit much to read for having just finished a tutorial with a lot of steps.
3. Am I going to want to click to another page after going through all that?
4. What do the terms in the headings mean to me as a brand-new user?

For 1-3, perhaps consider condensing this down to just the necessities and adding it to the existing **Conclusion** section at the end of the tutorial - that's currently [here](https://docs.dagster.io/guides/dagster/asset-tutorial/testing-assets#conclusion). We can always move that section to another page if a different step will be last in the new tutorial.

My thinking here is going to yet *another* page after completing a lengthy-ish tutorial might feel like a chore. If we add a section with a few short bullet points that's right at the end where the user already is, they don't have to go to another page to find ideas for what's next.

**For 4 (terms in headings)**, what would the headings/descriptions look like if we lead with what those features accomplish? Ex: **IO managers** > **Customize asset storage using IO managers**

Comment (erinkcochran87) [reply docs/content/guides/dagster/asset-tutorial/learning-more.mdx:6]: I do have some thoughts on the actual copy in here as well, but I'll hold off until we nail down the approach to this page/info overall before adding those.

**Review by erinkcochran87 — COMMENTED** [review]
I like where this is going! I added the copy suggestions I mentioned before - take a look and let me know your thoughts.

File: docs/content/guides/dagster/asset-tutorial/testing-assets.mdx [hunk 2]
```diff
@@ -57,6 +57,13 @@ Dagster is written to make testing easy in a domain where it has historically be
...
+🎉 Congratulations! Having reached this far, you now have a working, tested group of software-defined assets.
+
+What if you want to do more?
+* The tutorial covered how to manually materialize assets. Dagster can also kick off materializations automatically: on fixed [schedules](/concepts/partitions-schedules-sensors/schedules) or when your [sensor](/concepts/partitions-schedules-sensors/sensor) says to.
+* The tutorial covered assets whose entire contents get re-computed and overwritten with every materialization. When assets are large, it’s common to [partition](/concepts/partitions-schedules-sensors/partitions) them, so that each run only materializes a single partition.
+* The assets in the tutorial were materialized as pickle files on the local filesystem. [IO managers](/concepts/io-management/io-managers) let you customize how and where assets are stored - e.g. as tables in Snowflake or Parquet files in S3.
+* The tutorial assumed that your only goal was producing assets. You can also build and run arbitrary [jobs], on top of the same [op]() and [graph]() abstractions that underlie assets.
```
[the remark this answers is not in the document]
Comment (sryza) [reply docs/content/guides/dagster/asset-tutorial/testing-assets.mdx:66]: yeah, exactly

**Review by erinkcochran87 — COMMENTED** [review]
Only comment is to capitalize the letter of the first word after the hyphens. Ex: `- this tutorial` to `- This tutorial`

Otherwise, looks great!

**Review by erinkcochran87 — APPROVED** [review]

**Pull request merged**